Proof reading of nearly all items and game systems

Agentfunfun opened the pull request:
All alchemy, armors, consumables, dynamics, effects, flying swords, headjewelries, jewelries, miscjewelries, necklaces, other, shield, undergarments, vehicles, weapons and weaponsrange items proofread and adjusted accordingly. Some have likely been missed.

Agentfunfun committed:
Proof reading of game

Several typos and grammatical adjustments

Ormael13 commented:
Are you able to put that all in one large commit then make them all seperate commits in 1 PR?

Agentfunfun committed:
Merge branch 'master-wip' into master-wip

Agentfunfun committed:
First batch of scenes proofreading

Ashlands, Battlefield and Beach areas and encounters proofread. Fixed own mistake in PlayerAppearance

Agentfunfun committed:
Merge branch 'master-wip' of https://github.com/Agentfunfun/CoCX into master-wip

Agentfunfun reviewed (commented):
Mass typos fixing
